guides/micronaut-cloud-oidc-oracle/metadata.json

```
@@ -0,0 +1,8 @@
+{
+  "base": "micronaut-cloud-oidc-base",
+  "title": "Authenticate Micronaut Application users with OpenID Connect and Oracle IDCS",
+  "intro": "Learn how to create a secured Micronaut Application and authenticate with OpenID Connect in Oracle Cloud.",
```

```suggestion
  "intro": "Learn how to create a secured Micronaut application and authenticate with OpenID Connect in Oracle Cloud.",
```

"application" is not a proper noun